core/digital-twin-lib/src/dispatchers/twin/create.ts

```diff
@@ -17,28 +17,28 @@
   the following URL: https://evan.network/license/
 */
 
-import { DigitalTwin, DigitalTwinOptions } from '@evan.network/api-blockchain-core';
+import { DigitalTwin, DigitalTwinOptions, Ipfs } from '@evan.network/api-blockchain-core';
 import { Dispatcher, DispatcherInstance } from '@evan.network/ui';
-import { getDomainName, ipfs, } from '@evan.network/ui-dapp-browser';
-import { Ipfs } from '@evan.network/api-blockchain-core';
+import { getDomainName, ipfs } from '@evan.network/ui-dapp-browser';
 
 const dispatcher = new Dispatcher(
-  `lib.digital-twin.${ getDomainName() }`,
+  `lib.digital-twin.${getDomainName()}`,
   'twinCreateDispatcher',
   1000000, // depends probably on plugins etc.
-  '_digital-twin-lib.dispatchers.twin.create'
+  '_digital-twin-lib.dispatchers.twin.create',
 );
 
+/* eslint-disable no-param-reassign */
```

Comment:
not that easy to create a local var, needs a deep copy. Also if we need the changes in the next dispatcher step. 
Maybe we can disable it globally for certain situations: 
- changing data[attribute] e.g. in dispatchers
- changing state[attribute] e.g. in Vuex store

rule would be;

```json
"no-param-reassign": [
	"error",
	{
	  "props": true,
	  "ignorePropertyModificationsFor": [
	    "state",
      "data"
	  ]
	}
]
```

Reply:
Also found this solution, but it would enable every data property to be overwritten. Maybe we should use then a more special naming like `dispatcherData`?